drivers/include/periph/spi.h

```diff
@@ -43,6 +43,19 @@
  *    configures the bus with specific parameters (clock, mode) for the duration
  *    of that transaction.
  *
+ * @section     sec_pm (Low-) power implications
+ *
+ * As SPI buses are shared peripherals and the interfaces implements a
+ * transaction based paradigm, we leverage this for the SPI peripherals power
+ * management. After calling spi_init(), the SPI peripheral **should** be
+ * completely powered off (e.g. through peripheral clock gating). It **should**
+ * subsequently only be power on and enabled in between spi_acquire() and
+ * spi_release() blocks.
+ *
```

Comment:
I guess my comments will all look similar: IMHO we need to find out the performance implications of this fine-grained power control.